portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,7 +13,11 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
```

Comment:
nit: keep imports in alphabetical order

Though it's not in the style guide, we don't generally import `*`. it's better to be explicit with imports.